src/engraving/rendering/engravingitempreviewpainter.cpp

```diff
@@ -57,7 +64,61 @@ void EngravingItemPreviewPainter::paintPreview(mu::engraving::EngravingItem* ele
     paintPreviewForItem(element, params);
 }
 
-void EngravingItemPreviewPainter::paintItem(mu::engraving::EngravingItem* element, PaintParams& params)
+/// Returns a drum note prepared for preview.
+std::shared_ptr<Chord> EngravingItemPreviewPainter::getDrumNoteForPreview(const Drumset* drumset, int pitch)
```

Comment:
Might it be better to put this method in a separate `percussionutilities` file in the notation module, because it is in a way only loosely related to the other methods in this class. (And when the old drumset panel is removed, the method can be moved to the percussion panel model classes.)